.github/workflows/ci.yml

```diff
@@ -99,17 +66,16 @@ jobs:
     - name: install nextest
       uses: taiki-e/install-action@nextest
     - uses: actions/checkout@v2
-    - name: cargo nextest run
+    - uses: extractions/setup-just@v1
+    - name: just test
       run: |
-        cargo nextest run \
-          --workspace \
-          --all-features
+        just test
```

Comment:
similarly we could replace this whole thing with `- run: just test`

Reply:
yaml scares me